docs/security/programmatically-lock-user-accounts.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Programmatically locking and unlocking user accounts
+description: Lock users to prevent them from signing in based on your own custom criteria.
+---
+
+# Programmatically locking and unlocking user accounts
+
+## Unlocking a user programmatically
+
+You can programmatically unlock a user using the [UnlockUser](https://clerk.com/docs/reference/backend-api/tag/Users#operation/LockUser) Backend API operation. 
+
+## Usecases
+
+### "Unlock button"
```

Comment:
Why don't we have code examples for these? Maybe not for the entire use case, but at least to demonstrate how to call the backend api endpoint.
There's no information on whether calling it requires parameters, what it returns, etc.